mount/mount_linux.go

```diff
@@ -169,31 +183,39 @@ func detectSystemd() bool {
 }
 
 // MakeMountArgs makes the arguments to the mount(8) command.
-// Implementation is shared with NsEnterMounter
-func MakeMountArgs(source, target, fstype string, options []string) []string {
+// sensitiveOptions is an extention of options except they will not be logged (because they may contain sensitive material)
+func makeMountArgs(source, target, fstype string, options []string, sensitiveOptions []string) (mountArgs []string, mountArgsLogStr string) {
 	// Build mount command as follows:
 	//   mount [-t $fstype] [-o $options] [$source] $target
-	mountArgs := []string{}
+	mountArgs = []string{}
+	mountArgsLogStr = ""
 	if len(fstype) > 0 {
 		mountArgs = append(mountArgs, "-t", fstype)
+		mountArgsLogStr += strings.Join(mountArgs, " ")
 	}
 	if len(options) > 0 {
-		mountArgs = append(mountArgs, "-o", strings.Join(options, ","))
+		mountArgs = append(mountArgs, "-o", strings.Join(append(options, sensitiveOptions...), ","))
+		// exclude sensitiveOptions from log string
+		mountArgsLogStr += " -o " + strings.Join(options, ",")
+		if len(sensitiveOptions) > 0 {
+			mountArgsLogStr += sensitiveOptionsRemoved
+		}
 	}
 	if len(source) > 0 {
 		mountArgs = append(mountArgs, source)
+		mountArgsLogStr += " " + source + " "
 	}
 	mountArgs = append(mountArgs, target)
+	mountArgsLogStr += " " + target + " "
 
-	return mountArgs
+	return mountArgs, mountArgsLogStr
 }
 
-// AddSystemdScope adds "system-run --scope" to given command line
-// implementation is shared with NsEnterMounter
-func AddSystemdScope(systemdRunPath, mountName, command string, args []string) (string, []string) {
```

Comment:
same comment about a public method

Reply:
Will reintroduce and add a sensitive version as well.